src/dso_api/sentry.py

```diff
@@ -0,0 +1,24 @@
+from typing import Any, Final, Optional
+from urllib.parse import urlparse
+
+import environ
+
+env = environ.Env()
+
+# The makers of sentry only have internal types (in `_types`)
+# those are not considered stable atm, so we define
+# our own aliasses for now.
+Event = dict[str, Any]
+Hint = dict[str, Any]
+
+
+SENTRY_BLOCKED_PATHS: Final[str] = env.str("SENTRY_BLOCKED_PATHS", "")
+
+
+def before_send(event: Event, hint: Hint) -> Optional[Event]:
+    """Filters events before they are sent to the Sentry server."""
+    blocked_paths = SENTRY_BLOCKED_PATHS.split(",")
```

Comment:
Deze kan je toch eenmalig bovenaan de file parsen? en al ontdoen van lege waarden.

Reply:
Hier `settings.SENTRY_BLOCKED_PATHS` gebruiken. Hoef je niet nogmaals de `env.Environ()` te initialiseren.

Reply:
Heb ik wel overwogen. Maar, deze `sentry.py` module wordt zelf geimporteerd in `settings.py`, dus circular import.

Reply:
Met een iets andere opzet en de lazy loading werkt dit nu.